src/analyze/gui/stackstreeview.cpp

```diff
@@ -0,0 +1,28 @@
+#include "stackstreeview.hpp"
+
+#include <QApplication>
+#include <QClipboard>
+#include <QKeyEvent>
+
+StacksTreeView::StacksTreeView(QWidget* parent)
+    : QTreeView(parent) {
+
+}
+
+void StacksTreeView::keyPressEvent(QKeyEvent* event) {
+    if(!selectedIndexes().isEmpty() && event->matches(QKeySequence::Copy)) {
```

Comment:
can you instead add a `QShortCut` (probably with `Qt::WidgetWithChildrenShortcut` context) to the tree view and handle its `activated` signal and then fill the clipboard contents from a lambda? That should reduce the boilerplate code required for this feature